tests/generic/models.py

```diff
@@ -226,6 +226,12 @@ def save(self, *args, **kwargs):
         return super(ModelWithOverridedSave, self).save(*args, **kwargs)
 
 
+class ModelWithSaveKwargs(Dog):
+    def save(self, *args, **kwargs):
+        self.breed = kwargs.pop("breed")
+        return super(ModelWithSaveKwargs, self).save(*args, **kwargs)
```

Comment:
Can we use Python3-style super?

```pyrhon
return super().save(*args, **kwargs)
```

Reply:
Oh my, of course! Good note, somehow I simply copy-pasted from the other model, which I fixed as well in the last commit.

Thanks! 👍